public/js/index.js

```diff
@@ -0,0 +1,24 @@
+//Youtube slideshow for index page
+function onGoogleLoad() {
+  gapi.client.setApiKey('AIzaSyCzI5Gvpq1hPlfjvB81EVc0vLv_11eW7NI');
```

Comment:
Er what? JS is client side. If the JS can read the file, then so can everything else.

Although possibly not a bad idea to make it a constant variable though